interface/themes/labdata.css

```diff
@@ -0,0 +1,92 @@
+.graph {
+  color: var(--primary);
+}
+
+.graph_button {
+  background-color: #d3d7cf;
+  color: var(--black);
+  cursor: pointer;
```

Comment:
@stu01509 This file should not be here since I moved it...

Reply:
Update the commit.